packages/pharos/src/components/image-card/pharos-image-card.ts

```diff
@@ -245,9 +324,63 @@ export class PharosImageCard extends ScopedRegistryMixin(FocusMixin(PharosElemen
 
   protected override render(): TemplateResult {
     return html`<div class="card">
-      ${this._renderImage()} ${this._renderSourceType()}
-      <div class="card__title">${this.renderTitle} ${this._renderActionButton()}</div>
+      ${this._renderCheckbox()} ${this._renderImage()} ${this._renderSourceType()}
+      <div
+        class="card__title"
+        @mouseenter=${this._handleMouseEnterSelectable}
+        @mouseleave=${this._handleMouseLeaveSelectable}
+      >
+        ${this.renderTitle} ${this._renderActionButton()}
+      </div>
       ${this._renderMetadata()}
     </div>`;
   }
+
+  private _cardToggleSelect(event: Event): void {
+    if (this._isSelectableViaCard() || (event.target as Element)?.nodeName == 'PHAROS-CHECKBOX') {
+      // this is required to prevent navigation on the link click
+      event.preventDefault();
```

Comment:
I think would be better to remove the links from the card title and thumbnail when cards can be selected instead of overriding the link....

Reply:
I'm building confidence that leaving the links as is and using `event.preventDefault();` to avoid the link navigation is a good way to go. I feel that way because we want the styling and red on hover styling a link provides. Still not 100% convinced though. Very open to suggestions here!